ledger/src/lib.rs

```diff
@@ -40,6 +40,9 @@ extern crate eager;
 #[macro_use]
 extern crate solana_metrics;
 
+#[macro_use]
+extern crate lazy_static;
```

Comment:
Is https://doc.rust-lang.org/std/sync/struct.LazyLock.html more appropriate?

Reply:
Switched to `OnceLock`